### PR TITLE
Yessenia Rodriguez css-medical-canvas

### PR DESCRIPTION
**Pull request by yesseniarod**
Screen Shot 2021-01-24 at 8 58 44 PM
Screen Shot 2021-01-24 at 8 58 54 PM

### CODE REVIEW

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+              <a href="">&nbsp;&nbsp;Product&nbsp;&nbsp;</a>
+            </li>
+            <li>
+              <a href="">&nbsp;&nbsp;Implementation&nbsp;&nbsp;</a>
+            </li>
+            <li>
+              <a href="">&nbsp;&nbsp;About Us&nbsp;&nbsp;</a>
+            </li>
+            <li>
+              <a href="">&nbsp;&nbsp;Blog&nbsp;&nbsp;</a>
```
Comment (thebearingedge) [inline css-canvas-medical/index.html:33]: Try `padding` instead of spaces.

File: css-canvas-medical/styles.css [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+  margin: auto;
+  position: relative;
+  color: white;
+}
+
+h5 {
+  font-size: 15px;
+  position: relative;
+  bottom: -0.5em;
+}
```
Comment (thebearingedge) [inline css-canvas-medical/styles.css:253]: Try to use the box model instead of positioning for adjustments like these.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: css-canvas-medical/styles.css (hunk 2 above)
Comment (yesseniarod) [reply css-canvas-medical/styles.css]: Screen Shot 2021-01-31 at 3 21 00 PM

**Review by thebearingedge — APPROVED** [review]
Much better 👏

**Pull request merged**